src/test/compile-fail/issue-43733.rs

```diff
@@ -16,6 +16,8 @@ type Foo = std::cell::RefCell<String>;
 #[cfg(target_thread_local)]
 static __KEY: std::thread::__FastLocalKeyInner<Foo> =
     std::thread::__FastLocalKeyInner::new();
+//~^^ ERROR Sync` is not satisfied
+//~^^^ ERROR Sync` is not satisfied
```

Comment:
`check x86_64-pc-windows-gnu` failed, can't find these two errors.

Windows aren't `cfg(target_thread_local)` IIRC.

```
---- [compile-fail] compile-fail\issue-43733.rs stdout ----
	
error: C:/projects/rust/src/test/compile-fail/issue-43733.rs:17: expected error not found: Sync` is not satisfied
error: C:/projects/rust/src/test/compile-fail/issue-43733.rs:17: expected error not found: Sync` is not satisfied
error: 0 unexpected errors found, 2 expected errors not found
```

Reply:
Only mingw windows. MSVC windows does have `cfg(target_thread_local)`.
However... this error shouldn't be there, I think I forgot to put `#[thread_local]` on it when I wrote the test? Or maybe I wrote the test, then later changed `#[thread_local]` and forgot to update the test (because it didn't give these errors because MIR checks run far too late :/).

Anyway, just adding `#[thread_local]` should work.